spec/features/thredded/user_deletes_post_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+feature 'User deleting posts' do
+  scenario 'can delete their own post' do
+    user.log_in
+    topic = users_topic
+    topic.visit_topic
+    last_post = last_post(topic)
+
+    expect(last_post).to be_listed
+    expect(last_post).to be_deletable
+
+    last_post.delete
+
+    expect(last_post).to_not be_listed
```

Comment:
Total nitpick -- you can owe me a kick in the nuts the next time I see you for this, but -- can you change the `to_not`s to `not_to`s?

Haha I'm sorry! Almost the entire suite is using `not_to` so ... please please allow me that one little anal stupidity.